tastytaps/taps/models.py

```diff
@@ -30,3 +30,18 @@ class Price(models.Model):
 
     def __unicode__(self):
         return self.size
+
+
+class Taproom(models.Model):
+    name = models.CharField(max_length=200)
+    num_taps = models.PositiveSmallIntegerField(blank=True, null=True)
```

Comment:
I think we're going to need to require this since our taps display will use this.